### DISCUSSION TITLE
pull out motif coordinates

### DISCUSSION THREAD

**Discussion by yxiao832** [GreenleafLab/ArchR#766] [Questions / Documentation]
Hi ArchR team, 

From the peak-motif matrix, I am able to know the whether certain motif resides in certain peak. 

Is there a way to know the exact chromosome coordinates of a certain motif when it was annotated in a peak?  I want to do some mutation experiments on some annotated motifs.

Thank you!
Yang

**Comment by rcorces** [comment]
Please note that Issues are only for Bug reports! This is clearly stated on the issues report screen.
![image](https://user-images.githubusercontent.com/10343046/118563746-007f9900-b724-11eb-8e43-b6610ae342f4.png)

I am converting your issue to a Discussion topic which is the appropriate location.

**Comment by rcorces** [comment]
ArchR does not provide this functionality. However, I'm sure you can use motifmatchR to do so given outputs from ArchR. Good luck.
https://greenleaflab.github.io/motifmatchr/index.html

**Reply by yxiao832** [reply]
I am sorry I posted it at the wrong section. Thanks for the answer!
